packages/account/src/Sections/Profile/PersonalDetails/__tests__/input-group.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import InputGroup from '../input-group';
+
+describe('InputGroup', () => {
+    const children = 'This is children';
+    it('should render InputGroup with children passed inside', () => {
+        render(<InputGroup className='children_class'>{children}</InputGroup>);
```

Comment:
What is the significance of adding class_name in this test case? Is there a test case to check some specific conditional style?

Reply:
![Screenshot 2024-01-02 at 5 07 49 PM](https://github.com/binary-com/deriv-app/assets/103026762/88bd3350-d983-4515-b440-78b608694caa)
Typescript will throw error if we didn't include it

Reply:
Seems like InputGroup has classNAme as a required prop. Is that expected?

Reply:
I think it shouldn't be required. Will make the className optional then